# Configuration file for twist_command_randomizer node
twist_command_randomizer:
  ros__parameters:
    autonomous_vel_topic: "autonomous_vel"
    joy_vel_topic: "joy_vel"
    keyboard_vel_topic: "keyboard_vel"
    publish_rate: 5.0  # Hz

# Configuration file for movement_reader node
movement_reader:
  ros__parameters:
    cmd_vel_topic: "cmd_vel"
    cmd_type_topic: "cmd_type"